### PR TITLE
NATIVE-261: Add feedback to search

### PR DESCRIPTION
**Pull request by Schedulaar**

### CODE REVIEW

File: src/modules/categories/components/Categories.js [hunk 1]
```diff
@@ -152,30 +154,33 @@ class Categories extends React.Component<PropsType> {
...
       // first level, we want to display a table with all first order categories
 
-      return <Tiles tiles={this.getTileModels(children)}
-                    title={CityModel.findCityName(cities, category.title)}
-                    onTilePress={this.onTilePress}
-                    navigateToFeedback={this.navigateToFeedbackOfCategories}
-                    theme={theme}
-                    t={t} />
+      return <SpaceBetween>
+        <View><Tiles tiles={this.getTileModels(children)}
```
Comment (klinzo) [inline src/modules/categories/components/Categories.js]: Using newlines makes this much clearer imo, same for `</View>`

File: src/routes/extras/components/Extras.js [hunk 2]
```diff
@@ -61,20 +62,20 @@ class Extras extends React.Component<PropsType> {
 
     const feedbackItems = [
       new FeedbackVariant(t('feedback:contentOfCity', {city: cityTitle}), EXTRAS_FEEDBACK_TYPE),
-      ...extras.map(xtra => new FeedbackVariant(t('feedback:extra', {extra: xtra.title}), EXTRA_FEEDBACK_TYPE)),
+      ...extras.map(extra => new FeedbackVariant(t('feedback:extra', {extra: extra.title}), EXTRA_FEEDBACK_TYPE)),
       new FeedbackVariant(t('feedback:technicalTopics'), CATEGORIES_FEEDBACK_TYPE)
     ]
 
-    navigation.navigate('FeedbackModal', { isPositiveFeedback, feedbackItems })
+    navigation.navigate('FeedbackModal', {isPositiveFeedback, feedbackItems})
```
Comment (klinzo) [inline src/routes/extras/components/Extras.js]: Change your autolinting rule or whatever does this :D

File: src/routes/search/components/SearchFeedbackBox.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  render (): React.Node {
+    const {resultsFound, query, t, theme} = this.props
+    if (!resultsFound || query === this.state.boxOpenedForQuery) {
+      return <FeedbackBox theme={theme}>
+        <NothingFoundText theme={theme}>{t('feedback:nothingFound')}</NothingFoundText>
+        <NothingFoundFeedbackBox query={query} t={t} theme={theme} />
+      </FeedbackBox>
+    } else {
+      return <FeedbackBox theme={theme}>
+        <Button titleStyle={{color: theme.colors.textColor}}
```
Comment (klinzo) [inline src/routes/search/components/SearchFeedbackBox.js]: Why do you use that much inline styling here?

Comment (Schedulaar) [reply src/routes/search/components/SearchFeedbackBox.js]: Well I need to inject the stylesheet object into the `titleStyle` attribute (or below the `buttonStyle` attribute) which is a bit harder with styled-components...

File: src/routes/extras/components/Extras.js (hunk 2 above)
Comment (Schedulaar) [reply src/routes/extras/components/Extras.js]: I adjusted it :D

File: src/modules/categories/components/Categories.js (hunk 1 above)
Comment (Schedulaar) [reply src/modules/categories/components/Categories.js]: That's just taste I guess :P

Comment (klinzo) [reply src/modules/categories/components/Categories.js]: Really? You like it like that? :astonished: :astonished: But thanks ^^

**Pull request merged**